pkg/ha/ha_parser.go

```diff
@@ -6,10 +6,13 @@ package ha
 
 import (
 	"fmt"
+
 	promModel "github.com/prometheus/common/model"
 	"github.com/timescale/promscale/pkg/log"
+	multi_tenancy_write "github.com/timescale/promscale/pkg/multi-tenancy/write"
```

Comment:
any time you need to alias a pkg like this it's code smell. It's a sign that the package is badly named and/or organized. I think this should be "github.com/timescale/promscale/pkg/tenancy" and the authorizer should be WriteAuthorizer

Reply:
That is very true. However, i feel keeping respective authorizer in their own packages is better for readability.
I have changed `multi_tenancy_write` -> `mtw` and it looks more pleasing. 

Let me know if ore changes are required here.